e2e/performTests.ts

```diff
@@ -19,10 +19,9 @@ const testFilesContent = (commands: TestCommands, config: string, files: string[
     });
   });
 
-export default (label: string, sample: string): void => {
-  const config = `samples/${sample}`;
-
-  const commands = new TestCommands(`e2e/__tmp__/${label}`, "../../..");
+export default (config: string): void => {
+  const additionalParents = config.replace(/[^\/]+/g, "..");
```

Comment:
what is this for?

Reply:
we need to be able to locate files in project root, e.g.`fabrica.sh`. anyway, the code you mentioned is removed and the whole `relativeRoot` is simplified and encapsulated in `TestCommands`